src/sidebar/services/test/api-index.json

```diff
@@ -0,0 +1,84 @@
+{
+  "links": {
+    "profile": {
+      "read": {
+        "url": "https://example.com/api/profile",
+        "method": "GET",
+        "desc": "Fetch the user's profile"
+      },
+      "update": {
+        "url": "https://example.com/api/profile",
+        "method": "PATCH",
+        "desc": "Update a user's preferences"
+      }
+    },
+    "search": {
+      "url": "https://example.com/api/search",
+      "method": "GET",
+      "desc": "Search for annotations"
+    },
+    "group": {
+      "member": {
+        "add": {
+          "url": "https://example.com/api/groups/:pubid/members/:userid",
+          "method": "POST",
+          "desc": "Add the user in the request params to a group."
+        },
+        "delete": {
+          "url": "https://example.com/api/groups/:pubid/members/:userid",
```

Comment:
This endpoint only supports the (literal) value of `me` for `:userid`... I'm not sure if that is captured entirely here?